units/host_allocator.go

```diff
@@ -254,33 +255,35 @@ func (j *hostAllocatorJob) Run(ctx context.Context) {
 	const lowRatioThresh = float32(.25)
 	terminateExcess := distro.HostAllocatorSettings.HostsOverallocatedRule == evergreen.HostsOverallocatedTerminate
 	if terminateExcess && hostQueueRatio < lowRatioThresh && len(upHosts) > 0 {
-
-		var killableHosts, newCapTarget int
-		if hostQueueRatio == 0 {
-			killableHosts = len(upHosts)
-			newCapTarget = 0
-		} else {
-			killableHosts = int(float32(len(upHosts)) * (1 - hostQueueRatio))
-			newCapTarget = len(upHosts) - killableHosts
-		}
-
-		if newCapTarget < distro.HostAllocatorSettings.MinimumHosts {
-			newCapTarget = distro.HostAllocatorSettings.MinimumHosts
-		}
-		// rough value to prevent killing hosts on low-volume distros
-		const lowCountFloor = 5
-		if killableHosts > lowCountFloor {
-
-			drawdownInfo := DrawdownInfo{
-				DistroID:     distro.Id,
-				NewCapTarget: newCapTarget,
+		distroIsNotByHour := !cloud.UsesHourlyBilling(&upHosts[0].Distro)
```

Comment:
nit: can this instead be the positive statement instead of negative, like:
 ```go
func (j *hostAllocatorJob) drawdownOverallocatedDIstros(<stuff>) {
distroIsHourly = cloud.UsesHourlyBilling(&upHosts[0].Distro)
if distroIsHourly {
    return
}
the actual logic...
}
```